eclair-core/src/main/scala/fr/acinq/eclair/io/Switchboard.scala

```diff
@@ -212,14 +212,14 @@ object Switchboard {
     // We are interested in incoming HTLCs, that have been *cross-signed* (otherwise they wouldn't have been relayed).
     // If the HTLC is not in their commitment, it means that we have already fulfilled/failed it and that we can remove
     // the command from the pending relay db.
-    val channel2Htlc: Set[(ByteVector32, Long)] =
-    channels
-      .flatMap(_.commitments.remoteCommit.spec.htlcs)
-      .filter(_.direction == OUT)
-      .map(htlc => (htlc.add.channelId, htlc.add.id))
-      .toSet
-
-    val pendingRelay: Set[(ByteVector32, Long)] = relayDb.listPendingRelay()
+    val channel2Htlc: Seq[(ByteVector32, Long)] = for {
```

Comment:
I preferred the previous code, why did you change that? And this still should be a set imo.

Reply:
for aesthetic reasons :) somehow `for` seems more elegant to me, I'll change back.

Reply:
Thanks